scss/_local-navigation.scss

```diff
@@ -0,0 +1,131 @@
+.local-nav {
+  // scss-docs-start local-nav-css-vars
+  --#{$prefix}local-nav-padding-y: #{$local-nav-padding-y};
+  --#{$prefix}local-nav-color: #{$local-nav-color};
+  --#{$prefix}local-nav-bg: #{$local-nav-bg};
+  --#{$prefix}local-nav-hover-color: #{$local-nav-color};
```

Comment:
We should probably use/add a `$local-nav-hover-color`, even if it references `$local-nav-color`.